pkg/controller/dnszone/dnszone_controller_test.go

```diff
@@ -1078,6 +1075,7 @@ func assertDNSZoneConditions(t *testing.T, dnsZone *hivev1.DNSZone, expectedCond
 		if assert.NotNilf(t, condition, "did not find expected condition type: %v", expectedCond.Type) {
 			assert.Equal(t, expectedCond.Status, condition.Status, "condition found with unexpected status")
 			assert.Equal(t, expectedCond.Reason, condition.Reason, "condition found with unexpected reason")
+			assert.Equal(t, expectedCond.Message, condition.Message, "condition found with unexpected message")
```

Comment:
ntr: Added message matching, hence the expanded delta -- some of the message changes would have come about with the AWS SDK v2 migration.